code/modules/mob/living/carbon/human/human_attackalien.dm

```diff
@@ -37,13 +37,10 @@
 				visible_message("<span class='danger'>[M] has tackled down [src]!</span>", \
 					"<span class='userdanger'>[M] has tackled down [src]!</span>")
 			else
-				if (randn <= 99)

```

Comment:
Git does not let me post the comment where I want it to be, but you should change the `if (randn <= 80)` to a prob(80), as the number is now longer checked twice, and the disarm stance no longer has a third possible outcome.